src/graphql.js

```diff
@@ -61,12 +61,7 @@ function buildFundingEligibilityVariables() : FundingEligibilityParams {
     };
 }
 
-function getDefaultVariables<V>() : V {
-    // $FlowFixMe[incompatible-return]
-    return {};
-}
-
-export function callGraphQL<T, V>({ query, variables = getDefaultVariables(), headers = {} } : {| query : string, variables : V, headers? : { [string] : string } |}) : ZalgoPromise<T> {
+export function callGraphQL<T, V>({ query, variables = {}, headers = {} } : {| query : string, variables : V | { [string] : string }, headers? : { [string] : string } |}) : ZalgoPromise<T> {
```

Comment:
Yeah, that makes sense to me, I going to make the change

Reply:
The type `FundingEligibilityParams` was removed, incompatibility with that type and `mixed` in the value for variables. The removed type is internal we don't export it or the function `buildFundingEligibilityVariables `, I guess is save for us, since anyone else knows about that type.